http/http_client.hpp

```diff
@@ -763,9 +782,10 @@ class ConnectionPool : public std::enable_shared_from_this<ConnectionPool>
                 }
                 else
                 {
-                    BMCWEB_LOG_DEBUG << "Reusing existing connection "
-                                     << commonMsg;
-                    conn->doResolve();
+                    BMCWEB_LOG_ERROR << "Reusing existing connection "
```

Comment:
INFO: please. This is to identify this as information log and not an error

Reply:
How often does this log? We aren't going to be spamming the journal a crazy amount ?